.changeset/plenty-seahorses-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"@finos/legend-graph": patch
+---
+
+Resolve property mappings properly for `Inliine` and `Otherwise` SetImplementations.
```

Comment:
```suggestion
Properly resolve property mappings for `Inline` and `Otherwise` set implementations ([#650](https://github.com/finos/legend-studio/issues/650)).
```